west.yml

```diff
@@ -54,7 +54,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 15be5e615498377e9326cb2eb5819c7c62005299
+      revision: pull/648/head
```

Comment:
I think that this should be divided into two commits: one adding the macro and one updating the west.

Reply:
IMO we should keep it in a single commit to ensure that all commits will build all samples successfully. If we splint this into commit W and P where W does the manifest change and P does the flash_map_pm.h changes then a bisect landing on commit W will not be able to build some of the samples (nvs test sample if I remember correctly).

Reply:
Your macro is independent and may be provided prior to west update, additionally separate west.yml commit gives clear indication where Zephyr has been updated. I will not oppose if you choose to do both things with one commit, I just thing it looks cleaner in log.

Reply:
I see your point. I split it to two commits.